pico/tests/resources-pico/src/main/java/io/helidon/pico/tests/pico/tbox/impl/MainToolBox.java

```diff
@@ -41,6 +42,7 @@ public class MainToolBox implements ToolBox {
     @Inject
     @Preferred
     Provider<Hammer> preferredHammer;
+    private Provider<Hammer> setPreferredHammer;
```

Comment:
Why is there a change in visibility?
Also we do not want to support injection into private fields

Reply:
we inject via the setter method that is not private.

Reply:
Then the `@Inject` should be on the setter method, not on the field.

Reply:
it is already that way - I think you were reading the diff wrong.  I rearranged the fields to make it a little clearer.